contrib/setup

```diff
@@ -37,7 +37,7 @@ setup_run_wrappers()
     BIN=venv/bin/
     TEMPLATE=${BASE}/launch-venv.sh
     # launch wrappers
-    for F in fwupdtool fwupdmgr fwupd; do
+    for F in fwupdtool fwupdmgr fwupd fwupd-binder fwupdmgr-binder; do
```

Comment:
Maybe naive; but why even have a `fwupd` vs `fwupd-binder`?  IE if we're building for Android should we maybe ONLY do `fwupd`, but compile/link the `fu-binder-*`?

That is - I'm wondering if we really want a different `build` option.

```
option('build',
  type: 'combo',
  choices: [
    'all',
    'standalone',
    'library',
  ],
  value: 'all',
  description: 'build type',
)
```
Maybe `android` for a new one?  And then we will only compile the relevant pieces for android.

Reply:
that's a pretty good idea